packages/nimble-blazor/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -169,5 +169,22 @@ window.NimbleBlazor = window.NimbleBlazor ?? {
         getValidity: function (themeProviderReference) {
             return themeProviderReference.validity;
         }
+    },
+    WaferMap: {
+        getValidity: function (waferMapReference) {
+            return waferMapReference.validity;
```

Comment:
Have you manually verified this works? I'm kinda surprised you didn't need to pass it through as JSON like all the others below.

Reply:
I did not check, but I expected it to be similar with the table validity and the theme provider
![image](https://github.com/ni/nimble/assets/33986780/3fdb33af-206d-4957-a67a-6185c1aa2c0e)
will check right now

Reply:
Oh I forgot we have acceptance test infra in blazor! We should definitely make sure the code paths going through lib.Module are exercised like the getValidity method. Could running that be done in an acceptance test?